udata_front/theme/gouvfr/templates/reuse/card.html

```diff
@@ -1,27 +1,46 @@
+{% from theme('macros/organization_name_with_certificate.html') import organization_name_with_certificate %}
 {% cache cache_duration, 'reuse-card', reuse.id|string, g.lang_code %}
-<a
-    class="fr-raw-link unstyled w-100 fr-grid-row fr-grid-row--center"
-    href="{{ url_for('reuses.show', reuse=reuse, _external=True) }}"
-    title="{{ reuse.title }}"
->
-    <article class="reuse-card text-align-center fr-text--sm fr-my-0">
-        <img
-            class="reuse-image fr-mb-3v fr-responsive-img"
-            src="{{ reuse.image|placeholder('reuse', external=True) }}"
-            loading="lazy"
-            alt=""
-        />
-        <div>
-            {{ reuse.type_label }}
+<article class="fr-card fr-card--no-border fr-card--no-icon fr-enlarge-link">
+    <div class="fr-card__body">
+        <div class="fr-card__content fr-p-1w">
+            <h4 class="fr-card__title fr-text--md fr-m-0">
+                <a
+                    href="{{ url_for('reuses.show', reuse=reuse, _external=True) }}"
+                >
+                {{ reuse.title|mdstrip(55) }}
+                {% if reuse.private %}<span class="fr-badge fr-ml-1w">{{ _('Private') }}</span>{% endif %}
+                </a>
+            </h4>
+            <p class="fr-card__desc fr-my-1v not-enlarged">
+                {% if reuse.organization or reuse.owner %}
+                    {{ _(' by ') }}
+                    {% if reuse.organization %}
+                    <a
+                        href="{{ url_for('organizations.show', org=reuse.organization) }}"
+                    >
+                        {{organization_name_with_certificate(reuse.organization)}}
+                    </a>
+                    {% elif reuse.owner %}
+                    {{owner_name(reuse)}}
+                    {% endif %}
+                {% endif %}
+            </p>
+            <div class="fr-card__end fr-m-0 fr-p-0">
+                <p class="fr-card__detail fr-text--sm fr-m-0">
+                    {{ _('Published on %(date)s', date=reuse.created_at|dateformat(format='long')) }}
+                </p>
+            </div>
         </div>
-        <div class="text-grey-300">
-            {{ reuse.datasets|length or 0 }} {{ _('datasets') }}
+    </div>
+    <div class="fr-card__header">
+        <div class="fr-card__img">
+            <img
+                class="fr-responsive-img"
```

Comment:
Reuse images are stretched. Is this the desire behaviour. Ex:
![image](https://user-images.githubusercontent.com/28541902/174276186-397cc342-bf6b-4a8d-88ef-9f6a71ec3c32.png)

Reply:
No, I didn't notice this. I'll see how to fix it the DSFR way

Reply:
Image in card is no longer stretched. I choose a `contain` fit so users can see the all image. We could opt for `cover` if we want to force images to fill the entire space without stretch.